### ISSUE TITLE
[feature req.] more saved hotbars

### ISSUE THREAD

**Issue by Drigonis** [LukynkaCZE/better-saved-hotbars#10]
think the title gets the point across, having only 10 saved hotbars or however much it is (not entirely sure) is a bit limiting, especially if you want to have palettes and such saved, or custom NBT tools/weaponry. would be nice to add an option to configure the amount of available saved hotbars. obviously they wouldn't be able to be saved with the hotkey, but you could still drag and drop items into there.

**Comment by LukynkaCZE** [comment]
Yeah I had that in mind for a while, I'm just pretty busy with work

**Comment by VideoGameSmash12** [comment]
This could be implemented in two ways depending on if you want the hotbar files to be backwards compatible with vanilla Minecraft:
* The most vanilla-compatible way of doing so is to simply use a hotswap system and a mixin that changes what file gets loaded. That way, you don't have to worry about hotbar files being incompatible with vanilla Minecraft and thus prevents any potential issues in the long-term. I wrote a mod that did this, and it has proven to be incredibly reliable.
* If you don't care if the files are vanilla-compatible, you can obviously change how many rows exist in the code using a mixin [like this mod did](https://github.com/samipourquoi/more-toolbars/blob/master/src/main/java/io/github/samipourquoi/moretoolbars/mixin/MixinHotbarStorage.java#L22-L35), which would achieve the same thing but you wouldn't be able to access the items past a certain point in a vanilla client.
